KQ7365/orgamate-server
# Server side model construction

## Base models and routes constructed based upon project ERD: 
### `category.py`, `location.py`, `item.py`, `itemTag.py`, `task.py`, `priority.py`, `note.py`, `tag.py`

## URL routes initiated for `registration` and `login`:
###  `urls.py`

## Steps to test:
1) Pull down the `kq-createTodoModel` branch
2) step into branch and run cmd `code .`
3)activate `debugger` and ensure no errors
4) via `Postman`, enter address in URL http://localhost:8000/register
5) change tab to `POST`
6) ensure body meets the following fields:
{
      "email": "",
      "first_name": ",
      "last_name": "",
      "username": "",
      "password": ""
    }
7) Hit Send button and ensure you receive status code `201 created`
8) Lastly, take note of the Token in the response
